android/clientmanager/src/main/java/io/mosip/registration/clientmanager/service/MasterDataServiceImpl.java

```diff
@@ -196,25 +206,25 @@ public void manualSync() {
     @Override
     public void syncCertificate() {
         syncCertificate(() -> {
-        });
+        },"","","","");
     }
 
-    private void syncCertificate(Runnable onFinish) {
+    private void syncCertificate(Runnable onFinish,String APP_ID, String REF_ID, String SET_APP_ID, String SET_REF_ID) {
```

Comment:
can we rename the parameters here , they are confusing

Reply:
I have made changes